### PR TITLE
fix typo and some env vars

### PR DESCRIPTION
**Pull request by alphaolomi**

### CODE REVIEW

File: .env_sample [hunk 1]
```diff
@@ -1,12 +1,14 @@
 DJANGO_SECRET_KEY=django-secret-key
 
 DEBUG=True
-ALLOWED_HOSTS=.domain.com
-DB_NAME=database-name
-DB_USER=database-user
-DB_PASSWORD=database-password
+
+ALLOWED_HOSTS=127.0.0.1
```
Comment (ZendaInnocent) [inline .env_sample]: please add `ALLOWED_HOSTS=.localhost, 127.0.0.1`

**Review by ZendaInnocent — CHANGES_REQUESTED** [review]

File: .env_sample (hunk 1 above)
Comment (alphaolomi) [reply .env_sample]: ok

**Comment by alphaolomi** [comment]
done

**Comment by ZendaInnocent** [comment]
@alphaolomi thank you for your contribution

**Pull request merged**